src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -3,5 +3,9 @@ NuGet.Commands.IRestoreProgressReporter.EndProjectUpdate(string projectPath, Sys
 NuGet.Commands.IRestoreProgressReporter.StartProjectUpdate(string projectPath, System.Collections.Generic.IReadOnlyList<string> updatedFiles) -> void
 NuGet.Commands.RestoreArgs.ProgressReporter.get -> NuGet.Commands.IRestoreProgressReporter
 NuGet.Commands.RestoreArgs.ProgressReporter.set -> void
+NuGet.Commands.PackCollectorLogger.PackCollectorLogger(NuGet.Common.ILogger innerLogger, NuGet.ProjectModel.WarningProperties warningProperties, NuGet.Commands.PackCommand.PackageSpecificWarningProperties packageSpecificWarningProperties) -> void
+NuGet.Commands.PackCommand.PackageSpecificWarningProperties
+NuGet.Commands.PackCommand.PackageSpecificWarningProperties.PackageSpecificWarningProperties() -> void
+static NuGet.Commands.PackCommand.PackageSpecificWarningProperties.CreatePackageSpecificWarningProperties(System.Collections.Generic.IDictionary<string, System.Collections.Generic.HashSet<(NuGet.Common.NuGetLogCode, NuGet.Frameworks.NuGetFramework)>> noWarnProperties) -> NuGet.Commands.PackCommand.PackageSpecificWarningProperties
```

Comment:
The public API changes in your latest commit look accidental.  Please confirm.

Reply:
This one is ok, it happened because I resolved merge conflicts by merging to dev branch instead of rebasing the dev branch (with rebase if file has conflict on early commit, then it is very time consuming to resolve in all subsequent commits same), final compare you wouldn't see it.